server/tests/e2e/maps.e2e-spec.ts

```diff
@@ -2458,4 +2458,178 @@ describe('Maps', () => {
                 status: 401
             }));
     });
+
+    describe('GET /api/maps/{mapID}/ranks/{rankNumber}', () => {
```

Comment:
Even if on a certain level ranks/{blah} is the same endpoint (eh, it really isn't), lets split out the three variants into separate `describe` blocks.

Reply:
Or I guess two variants since we're not testing /friends. I've made a note to unit test that in the future.